saveModul/saveFile.js

```diff
@@ -1,15 +1,16 @@
 const mongoose = require('mongoose');
 const fs = require('fs');
 
+const Issue = require('../models/issueModel');
 const FacadeFileController = require('../facade/fileFacadeFunctions.js');
```

Comment:
Is this being used? Remember that controllers we are only accessing them from routes.